process_scenario_weo_2024.R

```diff
@@ -100,11 +99,14 @@ mpp_ats_raw <-
 logger::log_info("WEO 2024: Processing WEO 2024 data.")
 
 weo_2024 <-
-  pacta.scenario.data.preparation::prepare_weo_2024_scenario(
+  prepare_weo_2024_scenario(
```

Comment:
prefer explicit name spacing, e.g.
```suggestion
  pacta.scenario.data.preparation::prepare_weo_2024_scenario(
```